pkg/aux_/pool_participants.go

```diff
@@ -0,0 +1,19 @@
+package aux
+
+import (
+	"context"
+
+	restapi "github.com/interuss/dss/pkg/api/auxv1"
+	"github.com/interuss/stacktrace"
+)
+
+func (a *Server) GetDSSInstances(ctx context.Context, req *restapi.GetDSSInstancesRequest) restapi.GetDSSInstancesResponseSet {
+	if req.Auth.Error != nil {
+		resp := restapi.GetDSSInstancesResponseSet{}
+		setAuthError(ctx, stacktrace.Propagate(req.Auth.Error, "Auth failed"), &resp.Response401, &resp.Response403, &resp.Response500)
+		return resp
+	}
+
+	// Pool participant storage is not yet implemented.
+	return restapi.GetDSSInstancesResponseSet{Response200: &restapi.DSSInstancesResponse{DssInstances: &[]restapi.DSSInstance{}}}
```

Comment:
Probably not for this PR, but IMO it would be nice to have the code generator always generate a response 501 to be able to return a 501 for any call in such situations.

Reply:
I think that's in scope for this PR -- 501 is less misleading than success with an empty list.  Will update.